backend/README.md

```diff
@@ -1,12 +1,38 @@
 # Flotilla backend
 
+The backend of flotilla is created using ASP.NET.  
+Useful documentation of concepts and features in the .NET frameworks can be found 
+[here](https://docs.microsoft.com/en-us/dotnet/fundamentals/).
+
+## Setup
+To set up the backend on Windows/Mac, install visual studio and include the "ASP.NET and web development" workload during install.  
+If you already have visual studio installed, you can open the "Visual Studio Installer" and modify your install to add the workload.
+
+To set up the backend on linux..  
+CHRISTIAN PLEASE HELP ME  
+
+You will need the client secret of our Azure App Registration to be able to run the project locally.  
+This should be added in the
+[ASP.NET Secret Manager](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=linux#secret-manager)
+in the following format:
+```
+  "AzureAd": {
+    "ClientSecret": "ThisIsASecret"
+  }
+```
+
 ## Run
 
-To build and run the app, run the following command in the backend folder:
+To build and run the app, run the following command in the backend/api folder:
```

Comment:
We could consider writing:
```suggestion
To build and run the app, run the following command 'dotnet run --project api':
```
To have all the commands from the same folder. Thoughts?

Reply:
Sure!